test/collectionUtils/ClearMemberChoicesTransformer.test.ts

```diff
@@ -0,0 +1,33 @@
+import ClearMemberChoicesTransformer from "../../src/collectionUtils/ClearMemberChoicesTransformer";
+import MemberVariable from "../../src/model/dataTypes/MemberVariable";
+import SystemBoolean from "../../src/model/dataTypes/system/SystemBoolean";
+import SystemDate from "../../src/model/dataTypes/system/SystemDate";
+
+describe("ClearMemberChoicesTransformer", () => {
+  const transformer = new ClearMemberChoicesTransformer();
+  let member: MemberVariable;
+
+  beforeEach(() => {
+    member = new MemberVariable(
+      SystemBoolean,
+      "memberName",
+      false,
+      undefined,
+      false,
+      [SystemDate, SystemBoolean]
+    );
+  });
+
+  describe("#transform()", () => {
+    it("should clear the choiceTypes field", () => {
+      const result = transformer.transform(member);
+      expect(result).not.toBe(member);
+      expect(result.choiceTypes).toBeArrayOfSize(0);
```

Comment:
Codacy found an issue: [Unsafe call of an any typed value.](https://app.codacy.com/gh/MeasureAuthoringTool/model-info-parser/pullRequest?prid=5966907)